engine/src/main/resources/templates/ImageBuilder/dailyEmailTitleV2.txt

```diff
@@ -0,0 +1 @@
+Daily digest - Images - Red Hat Enterprise Linux
```

Comment:
We should have the full display name of the app here, like "Daily digest - Image builder - Red Hat Enterprise Linux"